homework15/src/ba/bitcamp/homework15/task01/Client1.java

```diff
@@ -0,0 +1,47 @@
+package ba.bitcamp.homework15.task01;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.OutputStreamWriter;
+import java.net.Socket;
+
+public class Client1 {
+
+	public static void main(String[] args) {
+		try {
+			Socket client = new Socket("localhost", 7897);
+			System.out.println("Connection established");
+
+			String fileAddress = "C:/BITCAMP/inheritance.docx";
```

Comment:
Izbjegavaj fiksanje ovako putanja do fajlova. Jer ako stavis ovo npr, sada na linux ili moj racunar nema sanse da radi (ovo za pocetak mozes koristiti, ali procitaj malo o java apsolutnim putanjama i relativnim putanjama). 
Evo za pocetak: http://www.avajava.com/tutorials/lessons/whats-the-difference-between-a-files-path-absolute-path-and-canonical-path.html